imports/api/rest/post-process-db-change-message.js

```diff
@@ -16,28 +18,82 @@ export default (req, res) => {
 
   const {
     notification_type: type,
-    invitee_user_id: inviteeId,
     case_title: caseTitle,
     case_id: caseId
   } = message
 
+  var emailAddr, emailContent, assigneeId, assignee
+
   switch (type) {
+    case 'case_new':
+      // https://github.com/unee-t/sns2email/issues/1
+      // When a new case is created, we need to inform the person who is assigned to that case.
+
+      assigneeId = message.assignee_user_id
+      assignee = Meteor.users.findOne({'bugzillaCreds.id': parseInt(assigneeId)})
+      if (!assignee) {
+        console.error('Could deliver message to missing user of BZ ID: ' + assigneeId)
+        return
+      }
+      emailAddr = assignee.emails[0].address
+      emailContent = caseNewTemplate(assignee, caseTitle, caseId)
+      try {
+        Email.send(Object.assign({
+          to: emailAddr,
+          from: process.env.FROM_EMAIL
+        }, emailContent))
+        console.log('Sent', emailAddr, 'notification type:', type)
+      } catch (e) {
+        console.error(`An error ${e} occurred while sending an email to ${emailAddr}`)
+      }
+
+      break
+
+    case 'case_assignee_updated':
+      // https://github.com/unee-t/sns2email/issues/2
+      // When the user assigned to a case change, we need to inform the person who is the new assignee to that case.
+
+      assigneeId = message.assignee_user_id
+      assignee = Meteor.users.findOne({'bugzillaCreds.id': parseInt(assigneeId)})
+      if (!assignee) {
+        console.error('Could deliver message to missing user of BZ ID: ' + assigneeId)
+        return
+      }
+      emailAddr = assignee.emails[0].address
+      emailContent = caseAssigneeUpdateTemplate(assignee, caseTitle, caseId)
+      try {
+        Email.send(Object.assign({
+          to: emailAddr,
+          from: process.env.FROM_EMAIL
+        }, emailContent))
+        console.log('Sent', emailAddr, 'notification type:', type)
+      } catch (e) {
+        console.error(`An error ${e} occurred while sending an email to ${emailAddr}`)
+      }
+
+      break
+
     case 'case_user_invited':
+
+      const {
+        invitee_user_id: inviteeId
+      } = message
+
       const invitee = Meteor.users.findOne({'bugzillaCreds.id': parseInt(inviteeId)})
       if (!invitee) {
         console.error('Could deliver message to missing user of BZ ID: ' + inviteeId)
         return
       }
-      const emailAddr = invitee.emails[0].address
-      const emailContent = emailTemplate(invitee, caseTitle, caseId)
+      emailAddr = invitee.emails[0].address
+      emailContent = caseUserInvitedTemplate(invitee, caseTitle, caseId)
```

Comment:
This line (and its 2 other equivalents) seems to be the only one with an actual difference between the 3 cases. You should define a single variable outside of the switch to store the desired template function and then just apply it and do everything else after the switch